src/post_process/m_global_parameters.fpp

```diff
@@ -691,6 +698,19 @@ contains
             buff_size = buff_size + fd_number
         end if
 
+        ! Configuring Coordinate Direction Indexes =========================
+        idwint(1)%beg = 0; idwint(2)%beg = 0; idwint(3)%beg = 0
```

Comment:
comment on what `idw` means?

Reply:
I updated the comments to include what the variable stands for instead  of just its meaning:
```fortran
! Cell Indices for the (local) interior points (O-m, O-n, 0-p).
! Stands for "InDices With BUFFer".
type(int_bounds_info) :: idwint(1:3)

! Cell Indices for the entire (local) domain. In simulation, this includes
! the buffer region. idwbuff and idwint are the same otherwise.
! Stands for "InDices With BUFFer".
type(int_bounds_info) :: idwbuff(1:3)
```